app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -82,4 +96,8 @@ public static void setLayoutMargin(@NonNull View view, float margin) {
       view.requestLayout();
     }
   }
+
+  private static boolean isRtlLayout(View view) {
+    return view.getLayoutDirection() == View.LAYOUT_DIRECTION_RTL;
```

Comment:
Per my earlier comment, shouldn't we be using ViewCompat to get the layout direction, instead?

Ditto in other files in this PR.

Reply:
Earlier it was using ViewCompat, due to which it has lint issues. I changed it to view and it fixed lint issues and removed suppression as per your comment  https://github.com/oppia/oppia-android/pull/3452#discussion_r668043999

Reply:
Earlier I was using ViewCompat and it was giving lint issues so changed it to View.

Reply:
I think your lint issues were because you weren't using the constants from ViewCompat itself (such as https://developer.android.com/reference/androidx/core/view/ViewCompat#LAYOUT_DIRECTION_RTL).